packages/core/src/balance/Balance.utils.ts

```diff
@@ -11,16 +11,41 @@
  */
 
 import { BN, formatBalance } from '@polkadot/util'
-
-import type { BalanceOptions } from '@kiltprotocol/types'
+import type {
+  BalanceNumber,
+  BalanceOptions,
+  MetricPrefix,
+} from '@kiltprotocol/types'
 
 export const KILT_COIN = new BN(1)
 
+export const Prefixes = new Map<MetricPrefix, number>([
+  ['femto', -15],
```

Comment:
Is there a way to avoid repeating the same names twice here and in `types/Balance.ts`?

Reply:
I would love to have that, but I think this wouldn't work with the compile/run time type existence in TS/JS.

If you have a suggestion, I'm more than happy to adopt that!

Reply:
Yeah I guess the only solution would be dynamic lookup from the entries of this map for anything that should be `MetricPrefix`.  I think Raphael is the wizard of those things, so I won't even try here 😂

Reply:
That'd be awesome, but maybe a lil itsybitsy tiny bit overkill for a measly 20ish elements. If your type list is 100s long, then that would be a must 😂